src/main/java/wolox/training/models/Book.java

```diff
@@ -110,6 +115,7 @@ public String getSubtitle() {
     }
 
     public void setSubtitle(String subtitle) {
+        Preconditions.checkNotNull(subtitle);
```

Comment:
The same as the previous comment.

Reply:
I put the messages and added it like constants.